package.json

```diff
@@ -49,6 +49,7 @@
     "webpack-dev-server": "^1.16.1"
   },
   "peerDependencies": {
+    "prop-types": "^15.3.2",
```

Comment:
I'm not actually sure if this should be a dependency or a peer dependency, but I put it here because react was here.